src/lib/net/classes/remote.ts

```diff
@@ -0,0 +1,65 @@
+/**
+ * !TODO: Add server/client differentation, yeah I know this should have made from the very first start, but lol,
+ * that will take a bit of my brain, but i already got something planned out for this.
+ */
+
+import { BaseEvent } from "../types";
+
+export declare enum EAction {
+	COND = "condition",
+	DO = "do",
+}
```

Comment:
Or alternatively, `C` and `D`, no need to make data bigger (it's a micro-op but it's just better

Reply:
Sounds good too, will take this into consideration.